enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -446,7 +447,13 @@ def signal_kernel(self, signum):
         else:
             raise RuntimeError("Cannot signal kernel. No kernel is running!")
 
-    def cleanup(self, connection_file=True):
+    def cleanup(self, restart=False):
```

Comment:
Actually, looking at the test failures, you'll need to change this signature back to what it was and call `cleanup_resources()` with the inverse parameter value...
```
======================================================================
ERROR: Verify authorized users can start a kernel, unauthorized users cannot
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/jupyter/enterprise_gateway/enterprise_gateway/tests/test_gatewayapp.py", line 102, in tearDown
    self.app.shutdown()
  File "/home/travis/build/jupyter/enterprise_gateway/enterprise_gateway/enterprisegatewayapp.py", line 302, in shutdown
    self.kernel_manager.shutdown_kernel(kid, now=True)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/notebook/services/kernels/kernelmanager.py", line 302, in shutdown_kernel
    return super(MappingKernelManager, self).shutdown_kernel(kernel_id, now=now)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/jupyter_client/multikernelmanager.py", line 211, in shutdown_kernel
    km.shutdown_kernel(now=now, restart=restart)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/jupyter_client/manager.py", line 395, in shutdown_kernel
    self.cleanup(connection_file=not restart)
TypeError: cleanup() got an unexpected keyword argument 'connection_file'
```

Reply:
Fixed